### PR TITLE
better error message when git is missing

### PR DESCRIPTION
**Pull request by camellia256**
1. I tried to use this with no `git` in `PATH` and got an ambiguous "FileNotFoundError". Adding a more direct error message should be helpful
2. I would like to use "raise .. from" to make the logic more clear, but that won't work in Python 2

### CODE REVIEW

**Comment by asottile** [comment]
thanks for the PR!

`FileNotFoundError` is new in python3+ so you'll need to use the generic `OSError` for python2.x's sake

Either a ` # pragma: no cover` or a quick test which mocks `subprocess` to always raise `OSError` will fix the coverage as well

File: tests/seed_isort_config_test.py [hunk 1]
```diff
@@ -324,3 +324,11 @@ def test_removing_file_after_git_add(tmpdir):
...
         expected = '[settings]\nknown_third_party=pre_commit\n'
         assert tmpdir.join('.isort.cfg').read() == expected
+
+
+def test_missing_git_from_path():
+    """expect user-friendly error message for a missing git"""
+    with pytest.raises(OSError,
+                       match='Cannot find git. Make sure it is in your PATH'):
+        with mock.patch('subprocess.check_output', side_effect=OSError):
+            main()
```
Comment (asottile) [inline tests/seed_isort_config_test.py]:
```python
with pytest.raises(OSError) as excinfo:
    with mock.patch.object(subprocess, 'check_output', side_effect=OSError):
        with tmpdir.as_cwd():
            _make_git()
            main(())
    msg, = excinfo.value.args
    assert msg == 'Cannot find ...'
```

Comment (camellia256) [reply tests/seed_isort_config_test.py]: thanks so much for your help!

Comment (asottile) [reply tests/seed_isort_config_test.py]: :tada:

**Comment by asottile** [comment]
thanks again!

**Pull request merged**